src/components/Modals/UseAsAliasModal/UseAsAliasModal.types.ts

```diff
@@ -0,0 +1,20 @@
+import { Dispatch } from 'redux'
+import { ModalProps } from 'decentraland-dapps/dist/providers/ModalProvider/ModalProvider.types'
+import { ENS } from 'modules/ens/types'
+
+export type Props = ModalProps & {
+  address?: string
+  isLoading: boolean
+  usedAsAlias: ENS[]
+  metadata: {
+    oldName: string
+    name: string
+  }
+  onSubmit: (address: string, name: string) => void
```

Comment:
This is fine but we try to type the methods we use with the "actual" actions they're calling. Like we did on `ENSListPage` !

```ts
// ENSListPage/ENSListPage.types.ts
import { openModal } from 'modules/modal/actions'

export type Props = {
  // (...)
  onOpenModal: typeof openModal
}
```

Reply:
Great I'll make this from now on ...